### ISSUE TITLE
document icons on Linux / possibly Windows too

### ISSUE THREAD

**Issue by mxa** [pure-data/pd-icon#7]
The document icon seems to be the based on a Mac OS specific style.
I was trying to figure out how to do this properly for Linux, considering all the different Themes out there it seems a bit of a mission-impossible.
Searching the web for "linux how to make document icon using the icon theme document template" led to nowhere.

**Comment by danomatika** [comment]
>The document icon seems to be the based on a Mac OS specific style.

Yes, as it is for macOS only, hence only created as an icon set (.icns extension). Pd does not currently have other platform specific file-type icons.
